typescript/sdk/src/token/checker.ts

```diff
@@ -25,6 +25,10 @@ export class HypERC20Checker extends HyperlaneRouterChecker<
   async checkChain(chain: ChainName): Promise<void> {
     await super.checkChain(chain);
     await this.checkToken(chain);
+    // We have adapted thi method to accept a proxyAdmin contract address parameter
```

Comment:
```suggestion
    // We have adapted this method to accept a proxyAdmin contract address parameter
```
Could you give a quick read of the comment's grammar too, it made it a little hard to understand quickly